src/platform/mhv/secondary-nav/containers/MhvSecondaryNav.jsx

```diff
@@ -5,6 +5,21 @@ const actionPrefix = 'MHV Secondary Nav';
 
 /**
  * MHV secondary navigation items. Note the first item is the home link.
+ *
+ * Notes:
+ * - The title is the text displayed in the navigation.
+ * - The abbreviation is an optional short version of the title, used for smaller screens.
+ * - The ariaLabel is paired with the abbreviation and it is used for accessibility purposes.
+ * - The actionName is used for analytics tracking.
+ * - The icon is the name of the icon to be displayed in the navigation.
+ * - The href is the link to the page.
+ * - The appRootUrl is paired with the href it is used to determine the root of the app for the navigation.
```

Comment:
This line almost gets to the core need of appRootUrl, which is that it can help determine which secondary nav element is the "current" one when the link actually points to a page within the app rather than the root URL.

That's hard to describe, and I don't think I captured it, but I think the "its for highlighting the active tab/app/section" is useful context